server/main/views.py

```diff
@@ -101,14 +99,12 @@ def post(self,request):
         UPSKILLING_TEMPLATE = "I want to know more about the career as a " 
         
         UPSKILLING_TEMPLATE += career + """
-        . First, I want to know exactly 2 main skills needed for my selected career. For each skill, I want exactly 2 specific suggestions as to where or how I can learn and develop it (example: website course, youtube video). I then want for each skill to know exactly a single way to verify or assess that certain skill that I learned (example: official certifications).
+        . First, I want to know at all main skills (as specific as possible) needed for my selected career, with description (200 word limit). For each skill, I want at all needed specific suggestions as to where or how I can learn and develop it (example: website course, youtube video), with description (200 word limit). I then want for each skill to know exactly a single way (as specific as possible) to verify or assess that certain skill that I learned (example: official certifications), with description (200 word limit).
```

Comment:
I think you need to exclude "at" before all, mainly on all "main skills" and all "specific suggestion"